Tangem/App/Utilities/TokenPriceFormatter/CommonTokenPriceFormatter.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CommonTokenPriceFormatter.swift
+//  Tangem
+//
+//  Created by skibinalexander on 21.05.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+struct CommonTokenPriceFormatter {
+    private let balanceFormatter = BalanceFormatter()
+}
+
+// MARK: - FeeFormatter
+
+extension CommonTokenPriceFormatter {
+    private var tokenPriceFiatFormattingOptions: BalanceFormattingOptions {
+        .init(
+            minFractionDigits: 2,
+            maxFractionDigits: 6,
+            formatEpsilonAsLowestRepresentableValue: false,
+            roundingType: .default(roundingMode: .plain, scale: 6)
+        )
+    }
+
+    func formatFiatBalance(_ value: Decimal?) -> String {
+        guard let value else {
+            return balanceFormatter.formatFiatBalance(value)
+        }
+
+        let fiatFormattingOptions: BalanceFormattingOptions = value > Constants.boundaryLowDigitOptions ? .defaultFiatFormattingOptions : tokenPriceFiatFormattingOptions
```

Comment:
только сейчас заметил, думаю уместно поставить вот так, чтобы если 0.01, то отображалось без знака `<`
```suggestion
        let fiatFormattingOptions: BalanceFormattingOptions = value >= Constants.boundaryLowDigitOptions ? .defaultFiatFormattingOptions : tokenPriceFiatFormattingOptions
```